Formula/skaffold.rb

```diff
@@ -14,7 +14,7 @@ class Skaffold < Formula
   end
 
   # relates to https://github.com/GoogleContainerTools/skaffold/pull/3775
-  # should be updated to use go after next release (> v1.5.0)
```

Comment:
Just curious why can it not move to go@1.14?

Reply:
we should! I'm going to update this - also we had to retag due to a bug in the release process, so I'll update the commit too